nafas/functions.py

```diff
@@ -30,6 +30,30 @@ def line(num: int = 70, char: str = "#") -> None:
     print(num * char)
 
 
+def is_int(number: Union[int, float]) -> bool:
+    """
+    Check that input number is int or not.
+
+    :param number: input number
+    """
+    if int(number) == number:
+        return True
+    return False
+
+
+def bpm_calc(program_data: Dict[str, Any]) -> float:
+    """
+    Calculate Breaths Per Minute (BPM).
+
+    :param program_data: program data
+    """
+    total_time_per_breath = sum(program_data["ratio"]) * program_data["unit"]
```

Comment:
`program_data` should be in the form of the following, right?
```json
{
    "name": "program1",
    "unit": 2,
    "pre": 3,
    "cycle": 10,
    "ratio": {
        "inhale": 2,
        "exhale": 2,
        "retain": 3,
        "sustain": 4
    }
}
```
Then `sum`ing it would sum the keys not the values. So I think it will raise `TypeError: unsupported operand type(s) for +: 'int' and 'str'`. Can you add some test for this functionality?

Reply:
I tested it and it's working ok, but not sure how robust it is, so I prefer doing `sum(program_data["ratio"]).values()` instead

Reply:
No, the correct format for the ratio is a list.

```python
{
    "name": "program1",
    "unit": 2,
    "pre": 3,
    "cycle": 10,
    "ratio": [2, 2, 3, 4]
    }
}
```